### PR TITLE
Moved icons over to svgs

### PR DESCRIPTION
**Pull request by markuskobler**
Removes font icons from the repository in favor of SVG icons.

### CODE REVIEW

File: public/assets/icons/common.svg [hunk 1]
```diff
@@ -21,4 +21,11 @@
...
   <symbol id="trash" viewBox="0 0 200 200"><path d="M33 59.1l11.1 137.6c.3 2.2 1.5 3.3 3.7 3.3h103.8c2.2 0 3.4-1.1 3.7-3.3l11.1-137.6H33zm51.3 115H70.4V84.8h13.9v89.3zm44.8 0h-13.9V84.8h13.9v89.3zM180.3 23.1c-.7-.7-1.6-1-2.7-1h-67V.4H89V22H22c-1.1 0-2 .3-2.7 1-.7.7-1 1.6-1 2.7v14.8c0 1.1.3 2 1 2.7s1.6 1 2.7 1h155.7c1.1 0 2-.3 2.7-1 .7-.7 1-1.6 1-2.7V25.8c0-1.1-.4-2-1.1-2.7z"/></symbol>
+  <symbol id="add" viewBox="111.48 114 120.96 120.96"><path d="M184.56 114h-25.2c-.747 0-1.353.233-1.82.7-.467.466-.7 1.073-.7 1.82v42.84H114c-.747 0-1.354.233-1.82.7-.467.467-.7 1.073-.7 1.82v25.2c0 .747.233 1.354.7 1.82.466.467 1.073.7 1.82.7h42.84v42.84c0 .747.233 1.354.7 1.82.467.467 1.073.7 1.82.7h25.2c.747 0 1.353-.233 1.82-.7.467-.466.7-1.073.7-1.82V189.6h42.84c.747 0 1.354-.233 1.82-.7.467-.466.7-1.073.7-1.82v-25.2c0-.747-.233-1.353-.7-1.82-.466-.467-1.073-.7-1.82-.7h-42.84v-42.84c0-.747-.233-1.354-.7-1.82-.467-.467-1.073-.7-1.82-.7z"/></symbol>
```
Comment (markuskobler) [inline public/assets/icons/common.svg]: So @seancrater has been ordering these lines alphabetically which I think is a good idea.

Comment (druiz) [inline public/assets/icons/common.svg]: You could do that in 3 commands with vim. How about emacs?

File: lib/Button.jsx [hunk 2]
```diff
@@ -2,12 +2,12 @@ import React from 'react';
...
 import './stylesheets/Button.scss';
+import Icon from './Icon';
 
-const renderIcon = (props) => {
-  const { icon } = props;
-
+const renderIcon = (icon) => {
   if (icon) {
-    return <i className="ss-icon">{ icon }</i>;
+    const style = { height: '20px' };
```
Comment (meagonqz) [inline lib/Button.jsx]: why not put this in a class?

File: public/assets/icons/common.svg (hunk 1 above)
Comment (seancrater) [inline public/assets/icons/common.svg]: Thanks @markuskobler, just makes it easier to read through IMO

**Pull request merged**